composer.json

```diff
@@ -145,9 +145,6 @@
             "drupal/entity_embed": {
                 "2511404 - Image entities/fields embedded using Entity Embed cannot be linked in CKEditor": "https://www.drupal.org/files/issues/entity_embed_links-2511404-31.patch"
             },
-            "drupal/datalayer": {
```

Comment:
why was this patch removed?

Reply:
This patch already added to the latest version of datalayer module